services/opencollective/opencollective-all.tester.js

```diff
@@ -20,43 +19,35 @@ t.create('renders correctly')
         contributorsCount: 276,
       })
   )
-  .expectJSONTypes(
-    Joi.object().keys({
-      name: 'backers and sponsors',
-      value: '35',
-      color: 'brightgreen',
-    })
-  )
+  .expectBadge({
+    label: 'backers and sponsors',
+    message: '35',
+    color: 'brightgreen',
+  })
 t.create('gets amount of backers and sponsors')
   .get('/shields.json')
-  .expectJSONTypes(
-    Joi.object().keys({
-      name: 'backers and sponsors',
-      value: nonNegativeInteger,
-    })
-  )
+  .expectBadge({
+    label: 'backers and sponsors',
+    message: nonNegativeInteger,
+  })
 
 t.create('renders not found correctly')
-  .get('/nonexistent-collective.json?style=_shield_test')
+  .get('/nonexistent-collective.json?style=_shields_test')
```

Comment:
Several of the OpenCollective tests were missing the `s` in `_shields_test`, the new helper highlighted this mistake.